python_modules/dagster/dagster/_core/storage/alembic/versions/033_add_asset_event_tags_table.py

```diff
@@ -0,0 +1,64 @@
+"""add_asset_event_tags_table
+
+Revision ID: 958a9495162d
+Revises: a00dd8d936a1
+Create Date: 2022-10-25 10:00:50.954192
+
+"""
+import sqlalchemy as db
+from alembic import op
+from sqlalchemy.dialects import sqlite
+
+
+from dagster._core.storage.migration.utils import has_index, has_table
+
+# revision identifiers, used by Alembic.
+revision = '958a9495162d'
+down_revision = 'a00dd8d936a1'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    if not has_table("asset_event_tags"):
+        op.create_table(
+            "asset_event_tags",
+            db.Column(
+                "id",
+                db.BigInteger().with_variant(sqlite.INTEGER(), "sqlite"),
+                primary_key=True,
+                autoincrement=True,
+            ),
+            db.Column("event_id", db.Integer, db.ForeignKey("event_logs.id", ondelete="CASCADE")),
+            db.Column("asset_key", db.Text, nullable=False),
+            db.Column("key", db.Text, nullable=False),
+            db.Column("value", db.Text),
+        )
+
+    if not has_index("asset_event_tags", "idx_asset_event_tags"):
+        op.create_index(
+            "idx_asset_event_tags",
+            "asset_event_tags",
+            ["asset_key", "key", "value"],
```

Comment:
I wonder if we need `id` in this index, since we're usually ordering by id

Reply:
https://github.com/dagster-io/dagster/pull/10056

Reply:
Hmm, so we should make this index `['asset_key', 'key', 'value', 'event_id']?